README.md

```diff
@@ -146,30 +146,34 @@ After that the scenario will keep running.
 
 ## Configuration
 
-amoc is configured through OTP application environmental variables that
-are loaded from the configuration file
+amoc is configured through OTP application environment variables that
+are loaded from the configuration file, operating system environment variables 
+(with prefix ``AMOC_``) and Erlang application environment variables.
 
 Internally, amoc is using the following settings:
 
 - ``interarrival`` - a delay in ms, for each node, between creating process
   for two consecutive users. Defaults to 50 ms.
-- ``repeat_interval`` - a delay in ms, for each user process that it waits
+- ``repeat_interval`` - a delay in ms, each user process waits
   before starting the same scenario again. Defaults to 1 minute.
 - ``repeat_num`` - number of scenario repetitions for each process.
   Defaults to ``infinity``.
 
 You can also define your own entries that you might later use in your
 scenarios.
 
-The ``amoc_config`` module, that is essentially a wrapper around functions
-manipulation the application's environment in the ``application`` module.
+The ``amoc_config`` is a module for getting environment variables. For each time
+we ask for a variable, ``amoc config`` firstly looks into OS environment variables
+(with ``AMOC_`` prefix e.g. if we want set ``interarrival`` we should set OS 
+environment variable ``AMOC_interarrival``), and if it doesn't find it gets value
+from Erlang application environment variables. What's more we can set variables 
+dynamically (via set OS variable or ``application:set_env(amoc, VARIABLE_NAME,
```

Comment:
Will the `application:set_env/3` have any effect if the `VERIABLE_NAME` is also set as OS environment variable?